archinstall/install.log

```diff
@@ -0,0 +1,54 @@
+Not enough permission to place log file at /var/log/archinstall/install.log, creating it in /home/anton/github/archinstall/archinstall/install.log instead.
```

Comment:
Drop this file from the PR and add an ignore for it

Reply:
Thought we had it already, will do :)